IP_POLICY_GUIDANCE.md

```diff
@@ -63,8 +63,9 @@ Each project must adopt either the [Developer Certificate of Origin (DCO)][DCO]
 or a Contributor License Agreement (CLA).
 
 The Board has pre-approved an [Individual CLA (ICLA)][ICLA]
-and a [Corporate CLA (CCLA)][CCLA]
-which projects can adopt without any further review.
+and a [Corporate CLA (CCLA)][CCLA].
+Project can adopt either the ICLA by itself or
```

Comment:
Either "A project can adopt ..." or "Projects can adopt ...". Singular "Project" reads a bit awkwardly here.

Reply:
Thanks for catching that typo.

Reply:
```suggestion
Projects can adopt either the ICLA by itself or
```